rest/src/main/resources/application.properties

```diff
@@ -1,6 +1,7 @@
 management.server.port=8080
 server.port=8080
 
+spring.profiles.active=dev
```

Comment:
this does not belong in the main appilcation.properties. this can have unintended side effects.

either set via maven run time or environment variable. not via application.properties

Reply:
Ive historically collapsed application.properties into one file that houses all of your runnable profiles.

But I can keep it going in separate files if we want to do something like application-dev.yaml

Reply:
https://reflectoring.io/spring-boot-profiles/

--> it is much easier to find properties for a certain profile an to compare them between profiles if they are split in different files. Also,  the reference manual says that the multi-document syntax can lead to unexpected behavior.

They  way you were doing it here, meant that you was not entirely correct.

if you wanted to use the multi-document file approach, you were correctly using the `spring.config.activate.on-profile=dev` property before I made a change.

however you were setting the active profile in the same file, which kind of defeats the purpose, as dev will always be active.

the better way is to put all the shared (and static) properties in the application.properties.

and then a file per environment.

running it with the correct profile:

`java -jar file.jar --spring-boot.run.profiles=production` or `mvn spring-boot:run -Dspring-boot.run.profiles=local` to set the correct profile. 

this has the additional benefit of setting the correct profile for the logbck.xml